src/razor/src/Completion/RazorCompletionItemProvider.ts

```diff
@@ -128,6 +130,8 @@ export class RazorCompletionItemProvider
                         completionItem.insertText = intellicodeCompletion.textEditText;
                     }
                 }
+
+                (<CompletionItem>completionItem).data = data;
```

Comment:
Does this override the data with the default in all cases, even if the completion item has provided its own?

Reply:
afaik we shouldn't be modifying this